nano/node/rpc.cpp

```diff
@@ -3314,6 +3314,43 @@ void nano::rpc_handler::unchecked_keys ()
 	response_errors ();
 }
 
+void nano::rpc_handler::unopened ()
+{
+	auto transaction (node.store.tx_begin_read ());
+	auto iterator (node.store.pending_begin (transaction, nano::pending_key (1, 0))); // exclude burn account
+	auto end (node.store.pending_end ());
+	nano::account current_account;
+	nano::uint128_t current_account_sum{ 0 };
+	while (iterator != end)
+	{
+		nano::pending_key key (iterator->first);
+		nano::account account (key.account);
+		nano::pending_info info (iterator->second);
+		if (node.store.account_exists (transaction, account))
+		{
+			if (account.number () == std::numeric_limits<nano::uint256_t>::max ())
+			{
+				break;
+			}
+			// Skip existing accounts
+			iterator = node.store.pending_begin (transaction, nano::pending_key (account.number () + 1, 0));
+		}
+		else
+		{
+			if (account != current_account)
+			{
+				response_l.put (current_account.to_account (), current_account_sum.convert_to<std::string> ());
+				current_account = account;
+				current_account_sum = 0;
+			}
+			current_account_sum += info.amount.number ();
+			++iterator;
+		}
+	}
+	response_l.put (current_account.to_account (), current_account_sum.convert_to<std::string> ()); // last one
```

Comment:
What if there are no unopened accounts? Could we make test for such case?

Reply:
Makes sense, see now